diff --git a/BesmokeInventoryApp.Server/Dtos/ProductDto.cs b/BesmokeInventoryApp.Server/Dtos/ProductDto.cs
index e9b4aa3b4abb09d36abe2ba31b5dc25ff52af24b..4fdcd65dcbab9fb00686720b02c197190bf5a9ec 100644
--- a/BesmokeInventoryApp.Server/Dtos/ProductDto.cs
+++ b/BesmokeInventoryApp.Server/Dtos/ProductDto.cs
@@ -1,11 +1,12 @@
 ﻿namespace BesmokeInventoryApp.Server.Dtos
 {
     public class ProductDto
     {
         public int Id { get; set; }
         public string Name { get; set; } = string.Empty;
         public string Type { get; set; } = string.Empty;
         public string Size { get; set; } = string.Empty;
         public string Material { get; set; } = string.Empty;
+        public int InitialQuantity { get; set; } = 0;
     }
 }
